.github/workflows/enhanced-testing.yml

```
@@ -423,17 +423,92 @@ jobs:
             echo "Found coverage directory: $coverage_dir"
             if [ -f "$coverage_dir/coverage-summary.json" ]; then
-              # Copy coverage summary to a predictable location
-              package_name=$(basename "$(dirname "$coverage_dir")" | sed 's/test-results-//' | sed 's/-.*//')
-              mkdir -p coverage-reports/$package_name
-              cp "$coverage_dir/coverage-summary.json" coverage-reports/$package_name/
-              echo "✅ Copied coverage report for $package_name"
+              # Extract package name from the full path
+              # Path structure: test-results/test-results-{package}-{run_id}-{name}/{package}/coverage/
+              # Try multiple extraction methods
+              package_name=""
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)